roles/kube_lvm_storage/defaults/main.yml

```diff
@@ -1,51 +1,13 @@
 debug: False
 
-storage_addon_dir: '{{ kube_config_dir }}/addons/storage_lvm'
+metalk8s_storage_addon_dir: '{{ kube_config_dir }}/addons/storage_lvm'
 
-# ################ #
-# LVM confguration #
-# ################ #
+# Set default LVm VGs storageclass and hostpath
+metalk8s_storageclass_vg_metalk8s: 'local-lvm'
+metalk8s_storageclass_default_vg_metalk8s: True
```

Comment:
There can be only on default SC, so this should be `metalk8s_default_storageclass: 'vg_metalk8s'` or similar, it's not  per-storage-class setting.

Reply:
Will change !